config/api.yml

```diff
@@ -877,6 +877,20 @@
         :identifier: service_tag
       - :name: unassign
         :identifier: service_tag
+  :settings:
+    :description: Settings
+    :identifier: ops_settings
+    :options:
+    - :collection
+    :methods: *70174834086080
+    :collection_actions:
```

Comment:
Settings implementation yes, but the role check on the api side differentiates it. ie. GET /api/settings checks collection role identifier, and GET /api/settings/product checks the resource role identifier.